<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QMainWindow" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Signup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-image: url(:/images/background/background_purple.png);}
QGroupBox {
border: 2px solid #aa55ff;
border-radius: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelWelcome">
    <property name="geometry">
     <rect>
      <x>452</x>
      <y>120</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 30pt &quot;Merriweather&quot;;</string>
    </property>
    <property name="text">
     <string>Welcome !</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>350</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 15pt &quot;Merriweather&quot;;
</string>
    </property>
    <property name="title">
     <string>Signup</string>
    </property>
    <widget class="QLabel" name="labelPassword">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>Password :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUsername">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>Username :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditUsername">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>112</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>152</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelError">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>350</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Merriweather&quot;;
color: rgb(255, 0, 0);
qproperty-alignment: 'AlignHCenter';</string>
     </property>
     <property name="text">
      <string>error</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShow">
     <property name="geometry">
      <rect>
       <x>266</x>
       <y>152</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/images/icon/eye_icon.png</normaloff>
       <normalon>:/images/icon/not_eye_icon.png</normalon>:/images/icon/eye_icon.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSignup">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>204</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:hover {
  background-color: rgb(165, 125, 235);
}
QPushButton {
	font: 10pt&quot;Merriweather&quot;;
	border-radius: 7px;
	background-color: rgb(180, 135, 255);
	border: 1px solid #000000;
	color: rgb(0, 0, 0);
}
QPushButton:pressed {
	background-color: rgb(135, 100, 195);
}</string>
     </property>
     <property name="text">
      <string>Signup</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFirstName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>32</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditLastName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>72</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
font: 9pt&quot;Merriweather&quot;;
border-radius: 7px;
background-color: rgb(215, 200, 255);
border: 1px solid #000000;
padding: 5px;
}
QLineEdit:hover {
	background-color: rgb(206, 192, 247);
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLastName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>Last Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFirstName">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Merriweather&quot;;</string>
     </property>
     <property name="text">
      <string>First Name :</string>
     </property>
    </widget>
    <zorder>lineEditPassword</zorder>
    <zorder>lineEditUsername</zorder>
    <zorder>labelError</zorder>
    <zorder>labelUsername</zorder>
    <zorder>labelPassword</zorder>
    <zorder>pushButtonShow</zorder>
    <zorder>pushButtonSignup</zorder>
    <zorder>lineEditFirstName</zorder>
    <zorder>lineEditLastName</zorder>
    <zorder>labelLastName</zorder>
    <zorder>labelFirstName</zorder>
   </widget>
   <widget class="QPushButton" name="pushButtonLogin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>230</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 10pt &quot;Merriweather&quot;;
	text-decoration: underline;
	color: rgb(0, 0, 0);
	background: transparent;
}
QPushButton:hover {
	color: rgb(68, 0, 102);
	font: 10.5pt &quot;Merriweather&quot;;
}
QPushButton:pressed {
	color: rgb(23, 0, 36);
}</string>
    </property>
    <property name="text">
     <string>Already Have An Account?</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelToDoList">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;Merriweather&quot;;</string>
    </property>
    <property name="text">
     <string>To Do List</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
